.github/workflows/e2e_test_ci_models.yml

```diff
@@ -6,10 +6,10 @@ on:
       - trunk
       - release-*
     paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'version.txt'
-      - 'acknowledgements.md'
+      - "docs/**"
```

Comment:
Fix your prettier settings to single quote :-)

Reply:
Configured - thank you